source/documentors/concepts/content_types.rst

```diff
@@ -1,49 +1,342 @@
+       - The document's structure could be more organized and easier to follow.
+       - The title needs to be more specific or is unrelated to the guide's topic.
+     - - The step sequence is unclear and difficult to follow.
+       - It doesn't have visual accompaniments.
+       - The document is not easy to follow and needs to be clarified for the reader.
 
-How-to document names should start with a verb to reflect action (e.g. ``create_new_course.rst``, ``schedule_course.rst``).
 
 .. _Reference:
 
-****************
-**Reference** 
-****************
+Reference
+*********
+
+Reference material is **information-oriented**. It can easily relate to **technical descriptions and factual information** about the system, APIs, parameters, etc. For example, “The Open edX Problem Types” or “The Open edX User Roles” would be good reference guides, as they are used by someone already familiar with the product and need to look up all the options it provides. 
+
+Reference material is like a map and details a function or feature of the Open edX platform. A map tells you what you need to know about the territory without having to go out and check it for yourself; a reference guide serves the same purpose for the product and its internal machinery.
+For example, many details about course subsections, such as the different publication states, grading configuration, and visibility, are not included in the how-to topic Create a Subsection but are fully described in the reference topic Course Subsections. These two topics are linked in the See Also sections.
+
+Reference Material Must Have:
+=============================
+
+#. **An accurate and precise description** of the product you’re referencing.
+
+#. **Provide examples of the uses and functions** of the product to make it more comprehensive.
+
+#. Generally, a reference **lists details or provides a glossary** of aspects of the product that the reader needs. (For example: definitions, commands, options, operations, features, flags, limitations, error messages, etc.)
+
+#. Provide **warnings** where appropriate.
+
+Technical Guidelines:
+=====================
+By following these recommendations, you will be able to write a good quality reference material:
+
+- Do nothing but describe. References have one job: **to explain** and do that **accurately and comprehensively**.
+- **Be accurate.** These descriptions must be accurate and kept up-to-date.
+- **Provide examples.** It is a valuable way of providing illustrations that help readers understand the references without becoming distracted from the job of describing them.
+- **The documentation structure should mirror the product's structure** so the user can work their way through it simultaneously. It doesn’t mean forcing the documentation into an unnatural structure. What’s important is that the logical, conceptual arrangement of and relations within the code should help make sense of the documentation.
```

Comment:
I puzzled over this sentence a bit. I think this is more accurate but please let me know if you meant something else.

```suggestion
- **The documentation structure should mirror the product's structure** so the user can work their way through it simultaneously. It doesn’t mean forcing the documentation into an unnatural structure. What’s important is that the logical, conceptual arrangement of and relations within the documentation should help make sense of the product.
```